OpenGov-OpenData/ckanext-showcase
# fix aria accessibility issue

This code fixes the accessibility issue caused by the `aira-describedby` attribute

To test this code:
- Make sure you have a web accessibility checking chrome extension, like [Axe](https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US),  installed 
- Install the code in the master branch 
- Open the CKAN homepage in Chrome
- Use the Axe extension to analyze the homepage, you shall notice a warning saying "[aria-*] attributes do not have valid values"
- Then install the code on this PR 
-  Use the Axe extension to analyze the homepage
-  the warning should be gone

* It also removes the redundant `slick.min.js` and `slick_min.js` file
